packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,257 @@
+        is required.</p>
+    </dd>
+
+    <dt>hasButton <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">buttonIcon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show an icon on the Button. Exists if
+        <code class="dummy-code">hasButton=true</code>.
+      </p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">buttonText <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show text on the Button. Exists if
+        <code class="dummy-code">hasButton=true</code>.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">buttonIconPosition <code>enum</code></dt>
+    <dd class="dummy-indent">
+      <p>
+        Positions the icon before or after the text. Exists if
+        <code class="dummy-code">hasButton=true</code>
+        and if
+        <code class="dummy-code">buttonText</code>
+        exists.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">
+          leading
+        </li>
+        <li>
+          trailing
+        </li>
+      </ol>
+    </dd>
+
+    <dt>hasLinkStandalone <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneIcon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show an icon on the Link (Standalone). Exists if
+        <code class="dummy-code">hasLinkStandalone=true</code>.
+      </p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneText <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show text on the Link (Standalone). Exists if
+        <code class="dummy-code">hasLinkStandalone=true</code>.
+      </p>
+    </dd>
+
+    <dt class="dummy-indent">linkStandaloneIconPosition <code>enum</code></dt>
+    <dd class="dummy-indent">
+      <p>
+        Positions the icon before or after the text. Exists if
+        <code class="dummy-code">haslinkStandalone=true</code>
+        and if
+        <code class="dummy-code">linkStandaloneText</code>
+        exists.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">
+          leading
+        </li>
+        <li>
+          trailing
+        </li>
+      </ol>
+    </dd>
+
+    <dt>hasDismiss <code>boolean</code></dt>
```

Comment:
[suggestion] I would call this `isDismissable` (it's a feature, not just a visual property) and add also a `onDismiss` hook

Reply:
I haven't brought it in yet, just on the docs https://github.com/hashicorp/design-system/pull/92, but makes sense. What sucks is it's `isDismissible` and hard to spell so maybe a better name can be had